client/src/components/subscription/createSubscription.js

```diff
@@ -0,0 +1,105 @@
+import React from 'react';
+
+/** TODO
```

Comment:
gotta add the colon for this guy

```suggestion
/** TODO:
```